src/osemgrep/reporting/Status_report.ml

```diff
@@ -67,12 +67,35 @@ let pp_status ~num_rules ~num_targets ~respect_git_ignore lang_jobs ppf =
           "<multilang>"
       | Xlang.L (l, _) -> Lang.to_lowercase_alnum l
     in
+    (* TODO: Improve Performance? *)
+    let lang_jobs =
```

Comment:
those are not lang_jobs anymore, but lang_stats, so better `let lang_stats = ...`